### PR TITLE
update tailwind.config for responsivness && add multi column layout w…

### PR DESCRIPTION
**Pull request by GrungeKarma**
…ith mobile responsiveness

standardized the mobile responsiveness in the tailwind.config added responsive div containers that add a multicolumn layout for nonmobile screens and keeps the single on mobile screens.

### CODE REVIEW

**Comment by GrungeKarma** [comment]
Just submitted an updated pull request. Sorry about the confusion.

**Review by RedFox0x20 — CHANGES_REQUESTED** [review]
Responsive to size changes and looks reasonable.

If implemented it would be a good idea to start looking at #38

There's only the one highlighted issue in `index.js`.
The Donation form also appears slightly squashed.

File: pages/index.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ const Cards = () =>
 
 export default function Home() {
   return (
-    <div className="md:container md:mx-auto">
+    <div className="w-screen">
```
[the remark this answers is not in the document]
Comment (GrungeKarma) [reply pages/index.js]: Removed the extraneous CSS.  I think the donation form size should be addressed with issue #38.

[the remark this answers is not in the document]
Comment (RedFox0x20) [reply pages/index.js]: Yes I think it would suit #38 too as more of a standardisation of panel sizes.

**Review by RedFox0x20 — APPROVED** [review]
All appears okay to me, the issue I previously mentioned is now fixed.

All seems well with reference to the comments above.

**Comment by audreyfeldroy** [comment]
Fantastic work @GrungeKarma!

I reviewed the diff in detail. It's extremely clean and elegant. The multi-column layout works very nicely, and you've gone above and beyond with implementing responsiveness too. 

Thank you for all your hard work. Your contribution is truly appreciated.

**Pull request merged**